src/app/item-triage/ItemTriage.tsx

```diff
@@ -55,321 +40,232 @@ export function doShowTriage(item: DimItem) {
-          title="Wishlists"
-          sectionId="triage-wishlist"
-          defaultCollapsed={false}
-          extra={wishlistItem ? <AppIcon className="thumbs-up" icon={thumbsUpIcon} /> : '–'}
-          disabled={!wishlistItem}
+      {item.bucket.inWeapons && <WishlistTriageSection item={item} />}
+      <LoadoutsTriageSection item={item} />
+      <FactorsTriageSection item={item} />
+      {item.bucket.inArmor && item.bucket.hash !== BucketHashes.ClassArmor && (
+        <ArmorStatsTriageSection item={item} />
+      )}
+    </div>
+  );
+}
+
+function WishlistTriageSection({ item }: { item: DimItem }) {
+  const wishlistItem = useSelector(wishListSelector(item));
+  const disabled = !wishlistItem;
+  const collapsedSetting = useSelector(collapsedSelector('triage-wishlist'));
+  const collapsed = disabled || collapsedSetting;
+  const [alreadyOpen] = useState(collapsed);
+
+  return (
+    <CollapsibleTitle
+      title={t('WishListRoll.Header')}
+      sectionId="triage-wishlist"
+      defaultCollapsed={false}
+      extra={wishlistItem ? <AppIcon className="thumbs-up" icon={thumbsUpIcon} /> : '–'}
+      disabled={disabled}
+    >
+      {wishlistItem?.notes?.length && (
+        <ExpandableTextBlock
+          linesWhenClosed={3}
+          className={popupStyles.description}
+          alreadyOpen={alreadyOpen}
         >
-          {wishlistItem?.notes?.length && (
-            <ExpandableTextBlock linesWhenClosed={3} className={popupStyles.description}>
-              <span className={popupStyles.wishListLabel}>
-                {t('WishListRoll.WishListNotes', { notes: '' })}
-              </span>
-              <span className={popupStyles.wishListTextContent}>{wishlistItem.notes}</span>
-            </ExpandableTextBlock>
-          )}
-        </CollapsibleTitle>
+          <span className={popupStyles.wishListTextContent}>{wishlistItem.notes}</span>
+        </ExpandableTextBlock>
       )}
-      <CollapsibleTitle
-        title="In Loadouts"
-        sectionId="triage-loadout"
-        defaultCollapsed={true}
-        extra={inLoadouts.length}
-        extraOnlyCollapsed
-        disabled={!inLoadouts.length}
-      >
-        <ul>
-          {inLoadouts.map((l) => (
+    </CollapsibleTitle>
+  );
+}
+
+function LoadoutsTriageSection({ item }: { item: DimItem }) {
+  const loadouts = useSelector(loadoutsSelector);
+  const inLoadouts = loadouts.filter((l) => l.items.some((i) => i.id === item.id));
+
+  return (
+    <CollapsibleTitle
+      title={t('Triage.InLoadouts')}
+      sectionId="triage-loadout"
+      defaultCollapsed={true}
+      extra={
+        <span className={styles.factorCollapsedValue}>
+          {inLoadouts.length}
+          <img src={helmet} className={styles.inLoadout} />
+        </span>
+      }
+      disabled={!inLoadouts.length}
+    >
+      <ul className={styles.loadoutList}>
+        {inLoadouts.map((l) => {
+          const edit = () => {
+            editLoadout(l, item.owner, {
```

Comment:
This edits loadouts in the context of the wrong store when the item is owned by a different character, e.g. I have a "Verity's + Headstone" Warlock loadout, but when I click the "edit" icon in Triage while the Eyasluna in the loadout is owned by my Titan, it hides all Warlock items from the loadout while editing (leading to missing armor and subclass).

Not sure if the fix should be here or down in the loadout drawer, but it's not super urgent because loadouts don't seem to change their class type or drop items when saving.

Reply:
fixed missing armor by editing in context of the vault. let me know if that seems likely to do something weird. it looks and save right, from simple testing

Reply:
#8344